fastn-js/js/dom.js

```diff
@@ -92,6 +92,7 @@ fastn_dom.property_map = {
     "font-style": "fst",
     "text-decoration": "td",
     "align-items": "ali",
+    "background-image": "bg",
```

Comment:
Write test for `fastn_dom.property_map` too in test.js. The values should be unique.

Reply:
I will do this in a separate PR